magmi/inc/magmi_config.php

```diff
@@ -37,7 +37,8 @@ public function getConfFile()
     public function getLastSaved($fmt)
```

Comment:
what is the type of $fmt? can you add a doc block while on it?

Reply:
mmm this is not my code so not sure what you want ,think who did that code should do that

Reply:
I know that its not your code, but you're changing line 40 `$lastsaved = date($fmt, filemtime($this->inifile));`
so you must know what type $fmt is, right?

Reply:
you mean strtime is not right so changed to date but the part inside are the same
but it from a list on php

Reply:
sorry it was strftime is been drop so it chage to date but the format are the same but i see what $fmt is doing 
as it could be wrong but it what was used to setup the strftime format
so it could be Y-m-d in date
am stilling learning php 8

Reply:
I have added the doc block as per tmotyl's request 

Also set a default for the string format to "r"

replacement code for the whole function  below - cbtrident can you adopt it please? 

```
/**
 * Get the file last saved date and time of a file
 *
 * @param string $fmt - any of the PHP time format strings documented here
 * https://www.php.net/manual/en/datetime.format.php 
 * 
 * A sensible one to use would be "r" producing a » RFC 2822/» RFC 5322 formatted date
 * Example of the returned sting: Thu, 21 Dec 2000 16:01:07 +0200
 *
 * @return string
 */    
    
    public function getLastSaved($fmt = "r")
    {
        if (file_exists($this->inifile)) {
            $lastsaved=date($fmt, filemtime($this->inifile));
        } else {
            $lastsaved="never";
        }
        return $lastsaved;
    }


```

Reply:
ok have add